src/session.rs

```diff
@@ -80,12 +81,14 @@ impl Session {
         let mut url_parser = UrlParser::new();
         url_parser.scheme_type_mapper(scheme_type_mapper);
         let url = try!(url_parser.parse(url_string));
+        let tls = if url.scheme == "amqps" { true } else { false };
```

Comment:
At least that one looks better than the former,
because to me it's like:
`if true { true } else { false}`